Python/pywarpx/picmi.py

```diff
@@ -168,6 +168,14 @@ class Species(picmistandard.PICMI_Species):
         during grid-based merging, with `velocity_grid_type == "cartesian"`. If
         a single number is given the same du value will be used in all three
         directions.
+
+    warpx_int_real_attributes: dict
+        Dictionary of extra integer particle attributes initialized with the
+        starting position and momentum for each particle.
+
+    warpx_add_real_attributes: dict
+        Dictionary of extra real particle attributes initialized with the
+        starting position and momentum for each particle.
```

Comment:
Maybe that is clearer to know which variables will be around as arguments?
```suggestion
        Dictionary of extra integer particle attributes initialized with a function of
        arguments (x,y,z,ux,uy,uz,t).

    warpx_add_real_attributes: dict
        Dictionary of extra real particle attributes initialized with a function of
        arguments (x,y,z,ux,uy,uz,t).
```

Reply:
I second this comment, though I might say "from an expression that is a function of the variables (x, y, z, ux, uy, uz, t)".